packages/manager/modules/vrack/src/add-dialog/index.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+
+import template from './template.html';
+
+const moduleName = 'ovhManagerVrackAddDialogModal';
+
+angular
+  .module(moduleName, ['oui', 'pascalprecht.translate'])
+  .component('vrackAddDialogModal', {
+    template,
+    bindings: {
+      onConfirm: '&',
+      onCancel: '&',
```

Comment:
```suggestion
      onConfirm: '<',
      onCancel: '<',
```
If you don't need to use variables on the callbacks, you can use '<' instead of '&'

Reply:
Since it's callback functions, one way data-binding is enough.